config/test/test.config.js

```diff
@@ -35,13 +35,20 @@ const pkgMap = fg
         };
     }, {});
 
-console.log(pkgMap);
+// NOTE: We need to use this plugin in order to turn the module exports
+// into module.exports. This will make it so that we can mock exports
+// correctly.
+const swcrc = JSON.parse(fs.readFileSync(path.join(root, ".swcrc"), "utf8"));
+(((swcrc.jsc ??= {}).experimental ??= {}).plugins ??= []).push([
```

Comment:
See https://github.com/magic-akari/swc_mut_cjs_exports 

The funky `??= {}` is a supported JS syntax that defaults an object if the key is undefined, BUT, it also ASSIGNS that default to the key. 

See [Nullish coalescing assignment (??=)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment)

Reply:
Fancy! Kind of hard to read, but I couldn't think of an alternative so I think I just need to retrain my brain to read this as-is; seems like a handy tool to have on the tool belt.

Reply:
Updated!